content-warehouse/src/main/java/contentwarehouse/v1/QuickStart.java

```diff
@@ -0,0 +1,142 @@
+      String parent = LocationName.format(projectNumber, location);
+
+      /* Create Document Schema with Text Type Property Definition
+       * More detail on managing Document Schemas: 
+       * https://cloud.google.com/document-warehouse/docs/manage-document-schemas */
+      DocumentSchema documentSchema = DocumentSchema.newBuilder()
+          .setDisplayName("My Test Schema")
+          .setDescription("My Test Schema's Description")
+          .addPropertyDefinitions(
+            PropertyDefinition.newBuilder()
+            .setName("stock_symbol")
+            .setDisplayName("Searchable text")
+            .setIsSearchable(true)
+            .setTextTypeOptions(TextTypeOptions.newBuilder().build())
+            .build()).build();
+
+      // Define Document Schema request
+      CreateDocumentSchemaRequest createDocumentSchemaRequest =
+          CreateDocumentSchemaRequest.newBuilder()
+            .setParent(parent)
+            .setDocumentSchema(documentSchema).build();
+
+      // Create Document Schema
+      DocumentSchema documentSchemaResponse =
+          documentSchemaServiceClient.createDocumentSchema(createDocumentSchemaRequest); 
+        
+
+      // Create Document Service Client Settings
+      DocumentServiceSettings documentServiceSettings = 
+          DocumentServiceSettings.newBuilder().setEndpoint(endpoint).build();
+      TextArray textArray = TextArray.newBuilder().addValues("GOOG").build();
+
+      // Create Document Service Client and Document with relevant properties 
+      try (DocumentServiceClient documentServiceClient =
+          DocumentServiceClient.create(documentServiceSettings)) {
+        Document document = Document.newBuilder()
+              .setDisplayName("My Test Document")
+              .setDocumentSchemaName(documentSchemaResponse.getName())
+              .setPlainText("This is a sample of a document's text.")
+              .addProperties(
+                Property.newBuilder()
+                  .setName(documentSchema.getPropertyDefinitions(0).getName())
+                  .setTextValues(textArray)).build();
+
+        // Define Request Metadata for enforcing access control
+        RequestMetadata requestMetadata = RequestMetadata.newBuilder()
+            .setUserInfo(
+            UserInfo.newBuilder()
+              .setId(userId).build()).build();
+            
+        // Define Create Document Request 
+        CreateDocumentRequest createDocumentRequest = CreateDocumentRequest.newBuilder()
+            .setParent(parent)
+            .setDocument(document)
+            .setRequestMetadata(requestMetadata)
+            .build();
+          
+        // Create Document
+        CreateDocumentResponse createDocumentResponse =
+            documentServiceClient.createDocument(createDocumentRequest);
+            
+        System.out.println(createDocumentResponse.getDocument().getName());
+        System.out.println(documentSchemaResponse.getName());
```

Comment:
What is the purpose of printing these values? What are you teaching the user?

Reply:
Mainly for the test, tried to follow a similar pattern to the DocAI sample
https://github.com/GoogleCloudPlatform/java-docs-samples/blob/main/document-ai/src/main/java/documentai/v1/QuickStart.java#L91